### PR TITLE
refactor: replace WaitOrTimeoutActor with wait_or_timeout function

### PR DESCRIPTION
**Pull request by DevSabb**
replace WaitOrTimeoutActor with wait_or_timeout function for the file chain/jsonrpc/jsonrpc-tests/tests/rpc_query.js

#6933

### CODE REVIEW

**Comment by DevSabb** [comment]
@matklad  Could you please take a look?

File: chain/jsonrpc/jsonrpc-tests/tests/rpc_query.rs [hunk 1]
```diff
@@ -10,14 +10,16 @@ use near_jsonrpc::client::{new_client, ChunkId};
...
+use near_network::test_utils::wait_or_timeout;
 use near_primitives::account::{AccessKey, AccessKeyPermission};
 use near_primitives::hash::CryptoHash;
 use near_primitives::types::{BlockId, BlockReference, EpochId, SyncCheckpoint};
 use near_primitives::views::QueryRequest;
 
 use near_jsonrpc_tests::{self as test_utils, test_with_client};
 
+pub type ControlFlow = std::ops::ControlFlow<()>;
+
```
Comment (matklad) [inline chain/jsonrpc/jsonrpc-tests/tests/rpc_query.rs:22]:
```suggestion
```

I don't think this type alias is needed, just a `use` seems to work fine?

Comment (DevSabb) [reply chain/jsonrpc/jsonrpc-tests/tests/rpc_query.rs:22]: I don't have any preference. I just followed the model PR. The `ControlFlow` is defined as type in runner.rs. 

https://github.com/near/nearcore/blob/7efb57a06906b297937fc0596139bd3da85aafc7/integration-tests/src/tests/network/runner.rs#L36

I am updating this PR with your suggestion though.

Comment (DevSabb) [reply chain/jsonrpc/jsonrpc-tests/tests/rpc_query.rs]: Fixed

**Review by matklad — APPROVED** [review]

**Pull request merged**